kitsune/announcements/tasks.py

```diff
@@ -30,7 +30,10 @@ def send_group_email(announcement_id):
 
     @safe_translation
     def _make_mail(locale, user):
-        subject = _("New announcement for {group}").format(group=group.name)
+        if groups.count() == 1:
+            subject = _(f"New announcement for {groups[0].name}")
+        else:
+            subject = _(f"New announcement for groups [{', '.join([g.name for g in groups])}])")
```

Comment:
Nit. Remove unmatched `)` at the end of the string.
```suggestion
            subject = _(f"New announcement for groups [{', '.join([g.name for g in groups])}]")
```